project.mml

```diff
@@ -642,15 +642,15 @@ Layer:
         (SELECT
             way,
             COALESCE((
-              'highway_' || (CASE WHEN highway IN ('residential', 'unclassified', 'pedestrian', 'service', 'footway', 'cycleway', 'track', 'path', 'platform') THEN highway END)),
+              'highway_' || (CASE WHEN highway IN ('pedestrian', 'service', 'footway', 'cycleway', 'track', 'path', 'platform') THEN highway END)),
               ('railway_' || (CASE WHEN (railway IN ('platform')
                               AND (tags->'location' NOT IN ('underground') OR (tags->'location') IS NULL)
                               AND (tunnel NOT IN ('yes', 'building_passage') OR tunnel IS NULL)
                               AND (covered NOT IN ('yes') OR covered IS NULL))
                               THEN railway END))
             ) AS feature
           FROM planet_osm_polygon
-          WHERE highway IN ('residential', 'unclassified', 'pedestrian', 'service', 'footway', 'track', 'path', 'platform')
+          WHERE highway IN ('pedestrian', 'service', 'footway', 'cycleway', 'track', 'path', 'platform')
```

Comment:
I see this adds back highway=cycleway, which is currently missing from this selection. 

Do we actually want to render highway=cycleway areas? 

Are there really areas where bicycles may travel freely in any direction?

Reply:
I haven't given it much thought either way. The WHERE clauses were the only places missing cycleway, we were attempting to render it in the MSS and the COALESCE